8/alpine/Dockerfile

```diff
@@ -34,6 +33,23 @@ RUN addgroup -g 1000 node \
     gpg --batch --keyserver hkp://ipv4.pool.sks-keyservers.net --recv-keys "$key" || \
     gpg --batch --keyserver hkp://pgp.mit.edu:80 --recv-keys "$key" ; \
   done \
+  && ( \
+    curl -fsSLO --compressed "https://unofficial-builds.nodejs.org/download/release/v$NODE_VERSION/node-v$NODE_VERSION-linux-$ARCH-musl.tar.xz" \
```

Comment:
Opened https://github.com/nodejs/unofficial-builds/issues/8 since there isn't an 8 release right now

Reply:
@LaurentGoderre there is an 8.16 available now 😄